src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -94,6 +95,9 @@ class GlobalSelectionHeader extends React.Component {
     // We should update store if there are any relevant URL parameters when component
     // is mounted
     if (Object.values(stateFromRouter).some(i => !!i)) {
+      if (!stateFromRouter.start && !stateFromRouter.end && !stateFromRouter.period) {
+        stateFromRouter.period = DEFAULT_STATS_PERIOD;
+      }
```

Comment:
I experimented doing this inside `getStateFromQuery()` but it caused problems in other pages.